### PR TITLE
[Nursery] CopDeepAR: Remove freq argument

### PR DESCRIPTION
**Pull request by rshyamsundar**
*Issue #, if available:*

Remove the `freq` argument in the overriding predictor.
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

File: src/gluonts/nursery/temporal_hierarchical_forecasting/train_and_eval_cop_deepar.py [hunk 1]
```diff
@@ -22,6 +22,9 @@
 )
 
 
+EVALUATE_ALL_LEVELS = True
```
Comment (jaheba) [inline src/gluonts/nursery/temporal_hierarchical_forecasting/train_and_eval_cop_deepar.py]: Is this change unrelated?

Comment (melopeo) [reply src/gluonts/nursery/temporal_hierarchical_forecasting/train_and_eval_cop_deepar.py]: In the previous version this parameter was not defined, and it was leading to an error.

Comment (melopeo) [reply src/gluonts/nursery/temporal_hierarchical_forecasting/train_and_eval_cop_deepar.py]: @rshyamsundar If we keep `EVALUATE_ALL_LEVELS = False` then I think we can merge this quickly and independently from the PR https://github.com/awslabs/gluonts/pull/2944.

**Review by melopeo — APPROVED** [review]
LGTM

**Pull request merged**